packages/amplify-provider-awscloudformation/src/aws-utils/aws.js

```diff
@@ -5,17 +5,20 @@ const configurationManager = require('../../lib/configuration-manager');
 
 aws.configureWithCreds = async (context) => {
   const httpProxy = process.env.HTTP_PROXY || process.env.HTTPS_PROXY;
+
   const config = await configurationManager.loadConfiguration(context, aws);
   if (config) {
     aws.config.update(config);
   }
 
   if (httpProxy) {
-    aws.config.update({
```

Comment:
this line shouldn't be removed, otherwise how is the aws configuration updated with the proxy?